### ISSUE TITLE
RMSE not matching

### ISSUE THREAD

**Issue by Mouradost** [lucktroy/DeepST#27]
Hey, I tried to compare my work with yours, i start by reproducing you code on tensorflow 1.10 however when I train your network most of the time i get an RMSE error = infinity but i run it few times and get some good results. For BikeNYC i get an RMSE error between 5.70% and 4.50% with an accuracy around 50% and for TaxiBJ the RMSE between 4.20% and 5.30% with an accuracy of 65%
I used the same hyperparameters as you with adam optimizer 
TaxiBJ : closeness =12 trend=1 period= 1 dropout 0.2 lr 0.005 number of layers 12 filter size 64 activation relu
BikeNYC: closeness =3 trend=4 period= 4 dropout 0.2 lr 0.005 number of layers 4 filter size 64 activation relu
I also set the seed to 1337
But I never get the same results as shown in your paper could you help me understand what is going wrong ? Is it because of the tensorflow version?
Thank you for your consideration
